strings/isisogram.go

```diff
@@ -13,10 +13,12 @@ import (
 	"strings"
 )
 
+type IsogramOrder int
```

Comment:
This doesn't need to be exported - your constants are exported and would work properly without it.

Reply:
IMO, the solution is to follow Go's standards and have an `isogram` package similar to the others (ie. `palindrome`). Thoughts?

Reply:
What do you mean? You don't need to export a `type` to export its `constants`. That is standard in Go, you can try looking at [this](https://go.dev/play/p/znzjqzxObwp).

Reply:
> we would normally also have the test file under the same package

No, that is why the `{package_name}_test` is allowed to live in the same directory, otherwise it won't compile if you change the package name to anything else.

Reply:
Ah, yes. That's where I got confused since my concern was with using that type with the struct defined for the tests. Apologies for not explaining that better and creating this back and forth. Appreciate your comments in this.